README.md

```diff
@@ -1,2 +1,28 @@
-This is a wrapper for Docker Credential helpers created by Azure Container Registry (ACR) team. This tool allows Azure Active Directory (AAD) based login. The credentials will be saved by the underlying cred-helpers configured at build time.
+# ACR Docker login helper
 
+This is a wrapper for Docker Credential helpers created by Azure Container Registry (ACR) team. This credential helper make use of Azure Active Directory (AAD) to obtain and maintain user's credentials.
```

Comment:
Please change to something like:

The ACR Docker Credential Helper allows users to sign-in to the Azure Container Registry service using their Azure Active Directory (AAD) credentials. This credential helper is in charge of ensuring that the stored credentials are valid, and when required it also renews the credentials for a repository.

For now, this credential helper works in tandem with the Azure CLI, which is required in order to initiate the credential flow. Once you've successfully logged in to your container registry with the Azure CLI, the credential helper administers the life cycle of your locally stored credential.